_partials/_devops-rest-api-get-started.md

```diff
@@ -0,0 +1,176 @@
+
+<Procedure>
+
+1. **Define the service creation payload in a JSON file**
+   ```bash
+   cat > service-config.json << 'EOF'
+   {
+     "name": "my-first-service",
+     "service_type": "TIMESCALEDB",
+     "region_code": "us-east-1",
+     "replica_count": 1,
+     "cpu_millis": 1000,
+     "memory_gbs": 4
+   }
+   EOF
+   ```
+
+1. **Submit service creation request**
+
+    1. Create the service using the POST endpoint:
+       ```bash
+       curl -X POST "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
+         -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
+         -H "Content-Type: application/json" \
+         -d @service-config.json
+       ```
+       $CLOUD_LONG creates a production environment for you. You see something like:
+       ```terminaloutput
+       {
+         "service_id":"asdfasdfasdf","project_id":"asdasdfasf","name":"my-first-service",
+         "region_code":"us-east-1", "service_type":"TIMESCALEDB",
+         "created":"2025-09-09T09:24:31.997767396Z", "paused":false,"status":"READY",
+         "resources":[{"id":"101240",
+         "spec":{"cpu_millis":1000,"memory_gbs":4,"volume_type":""}}],
+         "metadata":{"environment":"PROD"},
+         "endpoint":{"host":"oh.yeah.tsdb.cloud.timescale.com","port":123435},
+         "initial_password":"very-secret",
+         "ha_replicas":{"sync_replica_count":0,"replica_count":1}
+       } 
+       ```
+
+
+2. Save `service_id` from the response to a variable:
+   ```bash
+   # Extract service_id from the JSON response
+   export SERVICE_ID="service_id-from-response"
+   ```
+
+1. **Change the environment from production to development**
+
+  ```bash
+  curl -X GET "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services/${SERVICE_ID}" \
+    -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
+    -H "Content-Type: application/json" \
+    -d '{"environment": "DEV"}' 
+  ```
+You see something like:
+  ```terminaloutput
+  {                                          
+    "message": "Environment set successfully"
+  }
+  ```
+
+</Procedure>
+
+And that is it, you are ready to use the [$REST_LONG][rest-api-reference] to manage your
+$SERVICE_SHORTs in $CLOUD_LONG.
+
+## Security best practices
+
+Follow these security guidelines when working with the $REST_LONG:
+
+- **Credential management**
+    - Store API credentials as environment variables, not in code
+    - Use credential rotation policies for production environments
+    - Limit credential scope to minimum required permissions
+    - Never commit credentials to version control systems
+
+- **Network security**
+    - Use HTTPS endpoints exclusively for API communication
+    - Implement proper certificate validation in your HTTP clients
+    - Consider IP allowlisting for production API access
+    - Monitor API access logs for suspicious activity
```

Comment:
I don't think we currently give users the ability to do either of these things, either.